dashboard/db/migrate/20170306221907_merge_professional_learning_partner_into_regional_partner.rb

```diff
@@ -0,0 +1,34 @@
+class MergeProfessionalLearningPartnerIntoRegionalPartner < ActiveRecord::Migration[5.0]
+  def up
+    ActiveRecord::Base.record_timestamps = false
+    begin
+      ProfessionalLearningPartner.all.each do |plp|
+        regional_partner = RegionalPartner.find_or_create_by!(name: plp.name)
+        regional_partner.contact_id = plp.contact_id
+        regional_partner.urban = plp.urban
+        regional_partner.save!
+      end
+    ensure
+      ActiveRecord::Base.record_timestamps = true
+    end
+
+    drop_table :professional_learning_partners
```

Comment:
We can't drop this table yet, as it's still referenced in [Pd::Workshop.professional_learning_partner](https://github.com/code-dot-org/code-dot-org/blob/03221f605b8e1199af3390be508fd9ad8c6a04b4/dashboard/app/models/pd/workshop.rb#L378-L380), which is itself used in [payments](https://github.com/code-dot-org/code-dot-org/blob/2fbf38108b1b30016b85a67c328eae0666a2e72f/dashboard/lib/pd/payment/payment_calculator_base.rb#L63-L72), and in the [workshop dashboard controller](https://github.com/code-dot-org/code-dot-org/blob/26487387ba81a44764c472c44e3a8bfae50347ef/dashboard/app/controllers/pd/workshop_dashboard_controller.rb#L13).

In fact, I expect a bunch of unit tests to fail in circle because of this.